### DISCUSSION TITLE
Possible to have 1 Datasource to a local folder and use "path" option to select the .csv file?

### DISCUSSION THREAD

**Discussion by G0nz0uk** [grafana/grafana-csv-datasource#174] [Q&A]
Hello,

I copy all my csv files to the local below path from many sources:

/usr/share/csv/

This folder is on the Grafana server.  At the moment I'm created a new Datasource for each csv file and I was wondering if I can have just 1 datasource pointing to this local folder and then in Grafana use the "path" option just to choose the filename of the csv?

I did try the above but didn't work.

Thanks

**Comment by Jules-hexag** [comment]
I can do it as you described (one Datasource pointing to the folder) but using a path relative to the `grafana-xxx` binary you launched.
The csv filepaths ("path" option) also need to be relative to your Datasource.
If you put your `csvFolder` next to the binary, use `./csvFolder/` as your Datasource and enter as many "path" queries as you want (`./csv1.csv`).
